src/components/veNft/formControls/VeNftTokenSelectInput.tsx

```diff
@@ -0,0 +1,30 @@
+import { Trans } from '@lingui/macro'
+import { Form, FormInstance, Select } from 'antd'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import React, { useContext } from 'react'
+
+interface VeNftTokenSelectInputProps {
+  form: FormInstance
+}
+
+const VeNftTokenSelectInput = ({ form }: VeNftTokenSelectInputProps) => {
+  const { tokenAddress } = useContext(V2ProjectContext)
+  return (
+    <h3>
```

Comment:
**suggestion** for semantic juice, perhaps we could remove the wrapping `h3` and style the `label` accordingly instead?